pkg/agent/util/net_windows.go

```diff
@@ -59,6 +59,11 @@ const (
 
 	AntreaNatName = "antrea-nat"
 	LocalVMSwitch = "antrea-switch"
+
+	RT_FILTER_IF uint64 = 1 << (1 + iota)
```

Comment:
Could you add some comments for these definitions?

Reply:
Added.